### ISSUE TITLE
No resize signal carries the editor's size; only request-grid-update fires on resize

### ISSUE THREAD

**Issue by tilladam** [tilladam/slint-node-editor#6]
## Summary

Since 1.0.0 a resize of the editor is reported to the host only through `NodeEditorComputations.request-grid-update`. `viewport-changed(zoom, pan-x, pan-y)` fires on pan and zoom changes and carries no dimensions.

Before 1.0 (e.g. OlivierJG rev `8f6edbb`) `NodeEditor` had

```slint
changed width  => { NodeEditorComputations.viewport-changed(root.zoom, root.pan-x / 1px, root.pan-y / 1px); }
changed height => { NodeEditorComputations.viewport-changed(root.zoom, root.pan-x / 1px, root.pan-y / 1px); }
```

so a host that tracked the visible world rectangle from `viewport-changed` (reading the window size itself in the handler) kept working across resizes. In 1.0.x those handlers only call `request-grid-update`.

## Why it matters

A host that keeps its own notion of the visible viewport — for placing new nodes at the visible centre, for culling, for fitting — has no resize signal named for what it is. The only one is `request-grid-update`, which is named for a side effect; a host that draws no grid has no reason to connect it and silently loses resizes. We hit exactly that in QuAInt: after a plain window resize, nodes created afterwards landed at the old window's centre.

## Proposal

Either

- add `viewport-resized(/* width */ float, /* height */ float)` to `NodeEditorComputations` (or as a `NodeEditor` callback) and fire it from `changed width` / `changed height`, or
- extend `viewport-changed` with `width` and `height` and fire it on resize again.

The first keeps the existing signature stable; the second makes one callback describe the whole viewport. Either way the integration guide should list which callback a host must connect to learn about resizes.

## Version

slint-node-editor 1.0.1, Slint 1.18.0.
